admin/preprints/views.py

```diff
@@ -395,7 +401,7 @@ def delete(self, request, *args, **kwargs):
                     message='Confirmed SPAM: {}'.format(pid),
                     action_flag=CONFIRM_SPAM
```

Comment:
Nit: you could collapse this branching statement by setting action_flag value and the (non-formatted) message once you identify the action above